src/main/resources/application.yaml

```diff
@@ -35,8 +35,13 @@ spring:
       remove-abandoned: true
       remove-abandoned-timeout: 60
       log-abandoned: true
-      abandon-when-percentage-full: 0
-
+      abandon-when-percentage-full:
```

Comment:
Seem to have lost the value for `abandon-when-percentage-full`. Is that just github being weird?

Reply:
Hmm not sure but good spot!  Will add it back.